sonic_data_client/non_db_client.go

```diff
@@ -566,6 +566,7 @@ func (c *NonDbClient) Get(w *sync.WaitGroup) ([]*spb.Value, error) {
 		v, err := getter()
 		if err != nil {
 			log.V(3).Infof("PollRun getter error %v for %v", err, v)
+			return nil, err
```

Comment:
Would it make more sense here to `continue` instead of `return`?

Reply:
I think caller could determine whether retry based on return value of this function.